spec/rubocop/cop/style/align_parameters_spec.rb

```diff
@@ -504,7 +504,9 @@
                              '           b)',
                              'end'])
         expect(cop.offenses.size).to eq 1
-        expect(cop.offenses.first.to_s).to match(/method definition/)
+        expect(cop.offenses.map(&:to_s))
```

Comment:
The beginning of the fixed-indent context tests are autocorrections.  The only previous fixed indent test that expected offenses was a double indentation test that expected 3 offenses. This was also the first fixed indent test to have an assertion on the error message.  It was also a very relevant example to the problem this pull request addresses.